### PR TITLE
Typescript refactor + merge logos package

### PR DESCRIPTION
**Pull request by Gideonnn**
The logos package is now deprecated. Everything is merged inside of the sdk

### CODE REVIEW

File: scripts/build-logos.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+const productLogos = require('../public/data/product-logos.json');
+const { loadConfig, optimize } = require('svgo');
+const { appendFile, mkdir, readdir, readFile, writeFile, copyFile } = require('fs').promises;
+const camelcase = require('camelcase');
+const path = require('path');
+const babel = require('@babel/core');
+const { transform } = require('@svgr/core');
+const pluginTransformReactJsx = require('@babel/plugin-transform-react-jsx');
+
+const OUTPUT_DIR = path.join(__dirname, '../generated/logos');
```
Comment (mixplore) [inline scripts/build-logos.js:12]: Any reason we have `generated` dir instead of `dist`? 🤔  If not, can we switch it back?

File: scripts/build-logos.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  // Transform the svg to jsx
+  const component = await transform(data, { ref: true }, { componentName });
+
+  // Transform the jsx to js
+  const { code } = await babel.transformAsync(component, {
+    plugins: [[pluginTransformReactJsx, { useBuiltIns: true }]],
+  });
+
+  // const commonJsCode = code
+  //   .replace('import * as React from "react"', 'const React = require("react")')
```
Comment (mixplore) [inline scripts/build-logos.js:120]: Leftover commented code. Remove?

File: scripts/build-products.js [hunk 3]
```diff
@@ -120,33 +103,22 @@ const main = async () => {
...
-
-  console.log('Generating abi exports');
-  const abisPath = path.join(__dirname, '../dist/abis');
-  const abiExportsFile = path.join(__dirname, '../dist/abis/index.js');
-
-  const abis = fs
-    .readdirSync(abisPath)
-    .filter(file => file.endsWith('.json'))
-    .map(file => file.replace('.json', ''));
+  const destination = path.join(__dirname, '../generated');
```
Comment (mixplore) [inline scripts/build-products.js:106]: Why not `../dist`? :D

File: scripts/server.js [hunk 4]
```diff
@@ -1,12 +0,0 @@
...
-const path = require('path');
-const express = require('express');
-
-const assetsPath = path.join(__dirname, '../dist');
-const port = process.env.PORT || 3000;
-
-const app = express();
-app.use(express.static(assetsPath));
-
-app.listen(port, () => console.log(`Server started on port ${port}`));
```
Comment (mixplore) [inline scripts/server.js:12]: Why was this file deleted? 🤔

Comment (Gideonnn) [reply scripts/server.js:12]: We used to use an express server to host is. Now we host it on Cloudflare so we don't need it anymore 🎉

File: scripts/build-logos.js (hunk 1 above)
Comment (Gideonnn) [reply scripts/build-logos.js:12]: Yes, the generated folder is built before compiling, since importing from dist doesnt work (chicken and egg problem) so we get compile errors. The `generated` folder does not contain files that we should toch at all, since everything is generated automatically. The `public` folder is the opposite, all files in there are manually maintained.

**Review by mixplore — APPROVED** [review]
